content/soarian/dstu2/infrastructure/documentReference.md

```diff
@@ -73,7 +68,7 @@ Notes:
 
 #### Request with attachment
 
-	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc/DocumentReference?_format=json&attachment=docKey%3D101607B8CE55A4F20111E5B4F8D4AE526B7A48%26objNum%3D1%26created%3D20160301%26indexed%3D20160301&patientId=FaresRoni3425778"
+	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc/DocumentReference?_format=json&attachment=docKey%3D101607B8CE55A4F20111E5B4F8D4AE526B7A48%26objNum%3D1%26created%3D20160301%26indexed%3D20160301&patientId=FCC941D7-60B9-491D-BEED-27629E47CD4E 
```

Comment:
We have documented in detail about the 2 requests in the implementation notes. 

The API returns one C-CDA document per request. If a patient has more than one C-CDA document, then the first document is returned, and links for the remaining documents are provided in the response section “content.attachement.url”.

Subsequent calls, each made with the attachment parameter valued to a link provided in the first response, must be made to retrieve the remaining documents.

Please let us know if you have any questions on this.